usagov/vote-gov-drupal
# VOTE-3167 Update NVRF to v0.21.0

## Jira ticket
https://cm-jira.usa.gov/browse/VOTE-3167

## Description

Update the NVRF to [v0.21.0](https://github.com/usagov/vote-gov-nvrf-app/releases/tag/v0.21.0)

## Deployment and testing

### Post-deploy steps

1. lando retune

### QA/Testing instructions

1. Visit a state page and switch the language to Spanish.
2. Click on the NVRF button and fill out all fields (both required and not required).
3. Confirm all data is mapped correctly to the Spanish PDF.
4. Confirm the app is using the latest Spanish PDF (you will notice the dropdowns have been replaced with radio buttons as seen below)
![image](https://github.com/user-attachments/assets/e87c371e-1947-4b42-b629-301252fa3125)
5. Repeat steps 1-3 for the English app and PDF

## Checklist for the Developer




## Checklist for the Peer Reviewers
